docs/install/vendor-options.md

```diff
@@ -16,7 +16,7 @@ Here is a list of commercial Knative products (alphabetically):
 - [Google Cloud Run for Anthos](https://cloud.google.com/run/docs/gke/setup): <!-- TODO(ryangregg): add a description -->
 - IBM Kubernetes Service managed Knative add-on: <!-- TODO(duglin): add a description/link --> 
 - [Openshift Serverless](https://docs.openshift.com/container-platform/4.1/serverless/installing-openshift-serverless.html): <!-- TODO(bbrowning): add a description -->
-- [Pivotal Function Service](https://docs.pivotal.io/pfs): <!-- TODO(jurgen): add a description -->
+- [Pivotal Function Service (PFS)](https://docs.pivotal.io/pfs): is a platform for building and running functions, applications, and containers on Kubernetes. PFS is based on the riff open source project.
```

Comment:
Thanks @mattmoor - Could we link from `riff` to `https://projectriff.io` as well for more up-to-date downstream activity?

Reply:
done